PatternPal/PatternPal.Tests/Checks/MethodCheckTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿#region
+
+using static PatternPal.Core.Builders.CheckBuilder;
+
+#endregion
+
+namespace PatternPal.Tests.Checks;
+
+internal class MethodCheckTests
+{
+    [Test]
+    public void Method_Check_Accepts_Only_Classes()
```

Comment:
```suggestion
    public void Method_Check_Accepts_Only_Methods()
```
Be careful when copying other code, there is a lot of room to forget to rename things afterwards.